pinot-controller/src/main/resources/package.json

```diff
@@ -57,7 +57,8 @@
     "url-loader": "4.1.1",
     "webpack": "4.29.6",
     "webpack-cli": "3.3.0",
-    "webpack-dev-server": "3.2.1"
+    "webpack-dev-server": "3.2.1",
+    "hard-source-webpack-plugin": "0.13.1"
```

Comment:
What functionality does this package provide ?

Reply:
It was meant to speed up webpack build via caching but I've reverted it .